elixir - general, solution, two-fer remarks (#1344)
exercism/website-copy: automated-comments/elixir/solution/use_specification.md

<!--
This warning is raised when the solution doesn't have a type specification
-->

> Use a [type specification](https://elixir-lang.org/getting-started/typespecs-and-behaviours.html#types-and-specs) (`@spec`) to communicate the intended contract between function inputs and output.

Reviewer: ```suggestion
Use a [type specification](https://elixir-lang.org/getting-started/typespecs-and-behaviours.html#types-and-specs) (`@spec`) to communicate the contract between function inputs and output.
```

> - Elixir is a dynamically typed language, so types of data are only checked at run time, using type specification allows linters and static analysis like [`Dialyxir`](https://hexdocs.pm/dialyxir/readme.html) (based on Erlang's [Dialyzer](http://erlang.org/doc/man/dialyzer.html))

Reviewer: ```suggestion
- Elixir is a dynamically typed language, meaning that there are only type-checks at run-time, and using type specification enables the use of linters and static analysis like [`Dialyxir`](https://hexdocs.pm/dialyxir/readme.html) (based on Erlang's [Dialyzer](http://erlang.org/doc/man/dialyzer.html))
```
I don't quite understand what the _intended_ message was supposed to be, but the sentence felt incorrect to me. If this is what was meant, I would recommend writing it differently (or like the suggestion above).

Author: I was trying to give some context to the error and show that Elixir is intertwined with Erlang in a few regards.  I will try to reword.
